Unity-Technologies/Graphics
# Fix custom pass prefabs

## Purpose of this PR
Fix https://fogbugz.unity3d.com/f/cases/1244035/

Note that the PR includes more fixes than described in the bug report as the prefab integration wasn't working for the custom pass list either.

# Testing status
## Manual Tests
Manually tested to create custom pass volume, make it a prefab, and play with the different properties to see if it works as expected.
